src/app/public/modules/popover/popover-content.component.ts

```diff
@@ -215,6 +218,10 @@ export class SkyPopoverContentComponent implements OnInit, OnDestroy {
     this.horizontalAlignment = config.horizontalAlignment;
     this.placement = config.placement;
     this.popoverTitle = config.popoverTitle;
+    /* istanbul ignore else */
```

Comment:
We should be able to remove this ignore now that we have a test for this - correct?